src/touchstone/databases/elasticsearch.py

```diff
@@ -163,3 +168,32 @@ def access_dotted_field(self, input_dict, fields):
             else:
                 return None
         return tmp_dict
+
+    # def gen_result_list(self, response):
+    #    output_list = []
+    #    for hit in response:
+    #        output_list.append(hit.__dict__["_d_"])
+    #    return output_list
+
+    def get_timeseries_results(self, uuid, compute_map, index, identifier):
+
+        # not aggreated data
+
+        filters = compute_map.get("filter", {})
+
+        logger.debug("Initializing search object")
+        kw_identifier = identifier + ".keyword"
+        s = Search(using=self._conn_object, index=str(index)).query("match", **{kw_identifier: uuid})
+        # Apply filters
+        for key, value in filters.items():
+            s = s.filter("term", **{key: value})
+        logger.debug("Finished adding filters")
+        logger.debug("Built the following query: {}".format(json.dumps(s.to_dict(), indent=4)))
+
+        output_list = []
+        for hit in s.scan():
+            output_list.append(hit.__dict__["_d_"])
+        results_df = pd.DataFrame((d.to_dict() for d in s.scan()))
```

Comment:
I see something weird here, why are you scanning two times ES? can't we use the same response we already got?